cs-date-time/src/main/java/io/cloudslang/content/datetime/actions/GetCurrentDateTime.java

```diff
@@ -0,0 +1,86 @@
+import io.cloudslang.content.datetime.utils.Constants;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Created by stcu on 21.04.2016.
+ */
+public class GetCurrentDateTime {
+    /**
+     * Check the current date and time, and returns a java DateAndTime formatted string of it.
+     * If locale is specified, it will return the date and time string based on the locale.
+     * Otherwise, default locale will be used.
+     *
+     * @param localeLang    The locale language for date and time string. If localeLang is 'unix' the
+     *                      localeCountry input is ignored and the result will be the current UNIX
+     *                      timestamp. Examples:  en, ja, unix.
+     * @param localeCountry The locale country for date and time string. For example, US or JP.
+     *                      If localeLang is not specified, this input will be ignored.
+     * @return Current date and time.
+     */
+    @Action(name = "Get Current Date And Time",
+            outputs = {
+                    @Output(Constants.OutputNames.RETURN_RESULT),
+                    @Output(Constants.OutputNames.RETURN_CODE),
+                    @Output(Constants.OutputNames.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Constants.ResponseNames.SUCCESS, field = Constants.OutputNames.RETURN_CODE,
+                            value = Constants.ReturnCodes.RETURN_CODE_SUCCESS, matchType = MatchType.COMPARE_EQUAL,
+                            responseType = ResponseType.RESOLVED),
+                    @Response(text = Constants.ResponseNames.FAILURE, field = Constants.OutputNames.RETURN_CODE,
+                            value = Constants.ReturnCodes.RETURN_CODE_FAILURE, matchType = MatchType.COMPARE_EQUAL,
+                            responseType = ResponseType.ERROR, isOnFail = true)
+            })
+    public Map<String, String> execute(
+            @Param(value = Constants.InputNames.LOCALE_LANG, required = true) String localeLang,
+            @Param(value = Constants.InputNames.LOCALE_COUNTRY) String localeCountry) {
+
+        Map<String, String> returnResult = new HashMap<>();
+        Locale locale;
+        DateFormat formatter;
+        try {
+            Calendar calendar = Calendar.getInstance();
+
+            if (localeLang != null && !localeLang.isEmpty()) {
+                if ("unix".equals(localeLang)) {
+                    long timestamp = Math.round(calendar.getTimeInMillis()/1000);
+                    returnResult.put(Constants.OutputNames.RETURN_RESULT, "" + timestamp);
+                    returnResult.put(Constants.OutputNames.RETURN_CODE, Constants.ReturnCodes.RETURN_CODE_SUCCESS);
+
+                    return returnResult;
+                }
+
+                if (localeCountry != null && !localeCountry.isEmpty()) {
```

Comment:
AFAIK StringUtils.isNotBlank will do a supplementary validation against an successive blank spaces besides the null && empty validation. Will be more suitable here? WDYT?